### PR TITLE
Editors: get rid of deprecated in 20.x options

### PR DESCRIPTION
**Pull request by Zedwag**

### CODE REVIEW

File: js/ui/drop_down_box.js [hunk 1]
```diff
@@ -290,7 +290,7 @@ const DropDownBox = DropDownEditor.inherit({
             position: extend(this.option('popupPosition'), {
                 of: this.$element(),
             }),
-            onKeyboardHandled: opts => this.option('focusStateEnabled') && this._popupElementTabHandler(opts),
+            onKeyboardHandled: opts => focusStateEnabled && this._popupElementTabHandler(opts),
```
Comment (DokaRus) [inline js/ui/drop_down_box.js]: Will this function work correctly after changing the `focusStateEnabled` option at runtime?

File: js/ui/lookup.js [hunk 2]
```diff
@@ -652,7 +644,10 @@ const Lookup = DropDownList.inherit({
                 _fixWrapperPosition: false,
                 width: this._isInitialOptionValue('dropDownOptions.width')
                     ? (function() { return getOuterWidth(this.$element()); }).bind(this)
-                    : this._popupConfig().width
+                    : this._popupConfig().width,
+                height: this._isInitialOptionValue('dropDownOptions.height')
```
Comment (ksercs) [inline js/ui/lookup.js]: why? default height should not be equal to lookup input height

File: js/ui/lookup.js [hunk 3]
```diff
@@ -1036,28 +1031,10 @@ const Lookup = DropDownList.inherit({
...
-        });
-    },
-
     _optionChanged: function(args) {
-        const name = args.name;
-        const value = args.value;
+        const { name, fullName, value } = args;
 
-        switch(name) {
+        switch(fullName) {
```
Comment (ksercs) [inline js/ui/lookup.js]: I don't think it's a good idea: `name` is used in all other components

File: js/ui/lookup.js [hunk 4]
```diff
@@ -1129,6 +1090,12 @@ const Lookup = DropDownList.inherit({
             case 'cleanSearchOnOpening':
             case '_scrollToSelectedItemEnabled':
                 break;
+            case 'dropDownOptions.width':
```
Comment (ksercs) [inline js/ui/lookup.js]: why? When user set option to `auto` he expects option to get 'auto' value, not lookup element width

**Comment by ksercs** [comment]
@Zedwag 
let's also get rid of `popupHeight` in `dom_component.js`

File: js/ui/lookup.js (hunk 2 above)
Comment (Zedwag) [reply js/ui/lookup.js]: fixed

File: js/ui/lookup.js (hunk 4 above)
Comment (Zedwag) [reply js/ui/lookup.js]: https://github.com/DevExpress/DevExtreme/blob/21_2/testing/tests/DevExpress.ui.widgets.editors/lookup.tests.js#L1461

File: js/ui/lookup.js (hunk 3 above)
Comment (Zedwag) [reply js/ui/lookup.js]: ok

File: js/ui/drop_down_box.js (hunk 1 above)
Comment (Zedwag) [reply js/ui/drop_down_box.js]: fixed

**Comment by Zedwag** [comment]
> @Zedwag
> let's also get rid of `popupHeight` in `dom_component.js`

DropDownMenu still has `popupHeight` option

File: js/ui/lookup.js (hunk 4 above)
Comment (ksercs) [reply js/ui/lookup.js]: oh.. ok, let's leave it as it is..

File: js/ui/lookup.js [hunk 5]
```diff
@@ -1129,6 +1087,28 @@ const Lookup = DropDownList.inherit({
...
+                        });
+                        this._options.cache('dropDownOptions', this.option('dropDownOptions'));
+                        break;
+                    case 'dropDownOptions.height':
+                        this._popupOptionChanged({
+                            name,
+                            fullName,
+                            value: value === 'auto' ? this.initialOption('dropDownOptions').height : value
+                        });
+                        this._options.cache('dropDownOptions', this.option('dropDownOptions'));
```
Comment (DokaRus) [inline js/ui/lookup.js]:
```suggestion
                    case 'dropDownOptions.width':
                    case 'dropDownOptions.height':
                        this._popupOptionChanged({
                            name,
                            fullName,
                            value: value === 'auto' ? this.initialOption('dropDownOptions')[name] : value
                        });
                        this._options.cache('dropDownOptions', this.option('dropDownOptions'));
                        break;
```

Comment (Zedwag) [reply js/ui/lookup.js]: `name` equals 'dropDownOptions' here. I had idea to use something like `this.initialOption('dropDownOptions')[fullName.split('.')[1]]` but this looks realy weird :)

Comment (Zedwag) [reply js/ui/lookup.js]: implemented wia `getFieldName` utill

**Review by DokaRus — APPROVED** [review]

**Review by alexander-kotov-dx — APPROVED** [review]

**Pull request merged**